keras/saving/saving_lib.py

```diff
@@ -217,7 +251,7 @@ def save_weights_only(model, filepath):
     weights_store.close()
 
 
-def load_weights_only(model, filepath, skip_mismatch=False):
+def load_weights_only(model, filepath, sharded=False, skip_mismatch=False):
```

Comment:
Why should `sharded` be configurable here -- wouldn't it just depend on the file and the model?